sdk/identity/Azure.Identity/tests/AsyncLockWithValueTests.cs

```diff
@@ -131,7 +132,7 @@ public async Task AsyncLockWithValue_GetLockOrValueAsync_OneHundredCalls_HasNoVa
             var alwv = new AsyncLockWithValue<int>();
             var firstLock = await alwv.GetLockOrValueAsync(async);
             var tasks = new List<Task>();
-            for (var i = 0; i < 100; i++)
+            for (var i = 0; i < 50; i++)
```

Comment:
Do we need both the attribute and the values change?

Reply:
probably not, but the choice of 100 seems fairly arbitrary and increases the test duration.